src/components/space/ActionsChart.js

```diff
@@ -11,19 +11,65 @@ import {
   Bar,
   ResponsiveContainer,
 } from 'recharts';
-import { getActionsByDay, formatActions } from '../../utils/api';
+import {
+  getActionsByDay,
+  formatActions,
+  filterActionsByUser,
+  findYAxisMax,
+} from '../../utils/api';
 import { SpaceDataContext } from '../../contexts/SpaceDataProvider';
 
 const useStyles = makeStyles(() => ({
   typography: { textAlign: 'center' },
+  emptyChartAlert: {
+    display: 'flex',
+    justifyContent: 'center',
+    alignItems: 'center',
+    height: '450px',
```

Comment:
Is this used elsewhere? Constant?

Reply:
Yes, this is also the height of the ResponsiveContainer which houses BarChart in this same component. Factored out the 450 to a `CONTAINER_HEIGHT` constant in `/config/constants`